packaging/QueueNado.spec

```diff
@@ -28,8 +28,8 @@ cd 3rdparty
 unzip -u gtest-1.7.0.zip
 cd ..
 /usr/local/probe/bin/cmake -DCMAKE_CXX_COMPILER_ARG1:STRING=' -fPIC -Ofast -m64 -Wl,-rpath,. -Wl,-rpath -Wl,/usr/local/probe/lib -Wl,-rpath -Wl,/usr/local/probe/lib64 ' -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_CXX_COMPILER=/usr/local/probe/bin/g++
-make
-./UnitTestRunner
+make -j8
```

Comment:
If there is no reason for not doing it, we should make it "-j" not "-j8", right?